Backpack/Chip/Classes/BPKChip.swift

```diff
@@ -216,72 +186,69 @@ extension BPKChip {
         
         NSLayoutConstraint.activate([
             containerStackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: chipHorizontalSpacing),
-            trailingAnchor.constraint(equalTo: containerStackView.trailingAnchor, constant: chipHorizontalSpacing),
+            containerStackView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -chipHorizontalSpacing),
             containerStackView.topAnchor.constraint(equalTo: topAnchor, constant: chipVerticalSpacing),
-            bottomAnchor.constraint(equalTo: containerStackView.bottomAnchor, constant: chipVerticalSpacing)
+            containerStackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -chipVerticalSpacing)
```

Comment:
Can we flip these constraints? So we don't have to use the negative value? 
This is already done throughout the codebase

`bottomAnchor.constraint(equalTo: containerStackView.bottomAnchor, constant: chipVerticalSpacing)`